inference-engine/src/inference_engine/generic_ie.cpp

```diff
@@ -57,19 +57,14 @@ std::vector<InferenceEngine::IShapeInferExtensionPtr> ngraph::op::GenericIE::get
     return extensions;
 }
 
-ngraph::op::GenericIE::GenericIE(const ngraph::NodeVector& inputs,
-                                 const std::map<std::string, InferenceEngine::Parameter>& params,
-                                 const std::string type, const std::vector<PortIE>& outputs)
-    : GenericIE(as_output_vector(inputs), params, type, outputs) {}
-
```

Comment:
Why you removed this ctor?

Reply:
We have the same constructor which use vector of outputs.